### ISSUE TITLE
Question: RC Transmitter

### ISSUE THREAD

**Issue by JpEncausse** [TheDIYGuy999/RC_Transmitter#4]
Hi,
Sorry if my question is stupid, 

I'd like to setup an "Arduino Bridge" between an RC Car WLtoys K989 and a computer. 
Sending WebSocket/HTTP command to the Arduino then to the RC Car Receiver.
(Yes I know, lag will be a problem)

Should I start with RC_Transmitter repository code ? Or is there a lighweight version / article ?
Thanks

**Comment by jindaxia** [comment]
ESP8266 ?

**Comment by TheDIYGuy999** [comment]
There is no ESP8266 version, but one for STM32: https://github.com/TheDIYGuy999/RC_Transmitter_STM32

**Comment by Ahmedabdulamir** [comment]
Hi, I want to start building similar transmitter. Do I need Pro Mini 168 or 328 for the transmitter?. Also what about the receiver?

I like your work BTW, its so proffessional

Thanks

**Comment by TheDIYGuy999** [comment]
Hi,  you need the 328 version. Both the transmitter and the receiver use the same Atmel IC.

Also have a look at the manual in the transmitter section.

Thanks for the praise

**Comment by Ahmedabdulamir** [comment]
Thanks for you kind reply. Can I use 5v Pro mini instead of 3.3v version?

**Comment by jindaxia** [comment]
#5   YES！

**Comment by TheDIYGuy999** [comment]
Yes, but you have to make sure, that your battery voltage is at least 5.5V (before the 5V regulator) and that your radio module is supplied with 3.3V, using a separate regulator.

**Comment by jindaxia** [comment]
now I use 16M/3.3v， it‘s ok
I replaced the MIC5205 with a 3.3v version on the board， the MIC5205-3.3 is very easy to get，

**Comment by Ahmedabdulamir** [comment]
Thanks I got it. I think its best to use the 3.3v instead of using another regulator for the RF module. Thanks to all for your comments

**Comment by sosssego** [comment]
Can I use this to make 3 rc car/radios to work together?
